saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/EntityRelation/EntityRelationSensors.scala

```diff
@@ -30,7 +30,8 @@ object EntityRelationSensors {
   lazy val (sentencesAll, relationsAll, entitiesAll) = readConllData(path + "EntityMentionRelation/conll04.corp")
   lazy val (sentencesTrain, relationsTrain, entitiesTrain) = readConllData(path + "EntityMentionRelation/conll04_train.corp")
   lazy val (sentencesTest, relationsTest, entitiesTest) = readConllData(path + "EntityMentionRelation/conll04_test.corp")
-  lazy val (sentencesSmallSetTest, testRelationsSmallSetTest, entitiesSmallSetTest) = readConllData(resourcePath + "conll04-smallDocument.txt")
+  lazy val (sentencesSmallSet, testRelationsSmallSet, entitiesSmallSet) = readConllData(resourcePath + "conll04-smallDocument.txt")
+  lazy val (sentencesSmallSetTest, testRelationsSmallSetTest, entitiesSmallSetTest) = readConllData(resourcePath + "conll04_small.corp")
```

Comment:
Was adding a new document [`conll04_small.corp` necessary? I mean, `conll04-smallDocument.txt` is a also a small sample of the conll data. ](https://github.com/kordjamshidi/saul/tree/6686a062fcedbc3e162dcfde637dfa9607b1cd3c/saul-examples/src/test/resources/EntityMentionRelation)